ui/.eslintrc.json

```diff
@@ -90,9 +84,7 @@
           {
             "extendDefaults": true,
             "types": {
-              "{}": false,
-              "Object": false,
-              "Function": false
+              "{}": false
```

Comment:
i would ban empty object as well, otherwise, type safety is not guaranteed

Reply:
Yes I want to ban {}. but in this repo there are using many {} type. especially {} used in json rpc related type. this means if I remove {} in this repo, this will be super heavy task(Many typing will needed....). so I focused to ban Object and Function.

Reply:
Reducing `any` type and banning `{}`, applying strictNullChecks is very very important...
if I have time, I will address this kind little by little in other PR.

Reply:
i understand in terms of being a heavy task, so i approve